microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -45,122 +48,203 @@ As of Microsoft Edge 96, the following experimental features are turned on by de
 
-1.  Open the [Settings](../customize/index.md#settings) pane of DevTools.  To do this, select the **Settings** (gear) icon.
+1.  Click the **Settings** (gear) icon to open the [Settings](../customize/index.md#settings) pane of DevTools.
 
-1.  On the left side of the **Settings** pane, select the **Experiments** section.
+1.  On the left side of the **Settings** pane, click the **Experiments** section.
 
     :::image type="content" source="../media/experiments-devtools.msft.png" alt-text="The Experiments page in Settings" lightbox="../media/experiments-devtools.msft.png":::
 
-1.  On the **Experiments** page, scroll through the list of all available experimental features and select the checkbox next to each feature that you want to test.  Some experiments are turned on by default.
+1.  On the **Experiments** page, scroll through the list of all available experimental features. Click an empty checkbox to select each feature that you want to test. Some experiments are turned on by default.
+
+1.  Click a selected checkbox to turn off any feature you do not want to test.
 
-1.  Select the **X** in the upper right to close **Settings**.
+1.  Click the **X** in the upper right to close **Settings**.
 
-1.  Select the **Reload DevTools** button.
+1.  Click the **Reload DevTools** button.
 
 > [!NOTE]
-> Experimental features are constantly being updated and might cause performance issues.  To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.
+> Experimental features are constantly being updated and might cause performance issues. To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.
 
 
 <!-- ====================================================================== -->
+## Restore defaults and refresh
+
+You can restore the default Experimental features settings.
+
+1.  [Open DevTools](../open/index.md). Click the **Settings and more** button, which is an ellipsis (three dots). Then hover over **More tools**, and click **Developer tools**. Or press **Control + Shift + I**.
+ 
+1.  Click the **Settings** (gear) icon to open the [Settings](../customize/index.md#settings) pane of DevTools.
+
+1.  Click **Preferences** > **Restore defaults and refresh**.
+
+### Make sure default settings are restored
+
+1.  Open  **Settings** > **Experiments**.
+
+1.  Toggle a checkbox in **Experiments**, then close **Settings**.
+
+1.  View **Experiments** again and make sure your change persisted.
+
+1.  Close **Settings**.
+
+1.  Click **Settings** > **Preferences** > **Restore defaults and refresh**.
+
+1.  Open **Settings** > **Experiments** again to make sure see the experiment's checkbox reverted to the default state (selected or cleared).
```

Comment:
```suggestion
```
This content is useful for writers but possibly adds more confusion than help (pls assess), when skimming & reading the page.